README.md

```diff
@@ -21,6 +21,51 @@ _**Note**: below listed features are still under design and review phase and hen
 * New range functions
 * Freeze rows/columns.
 
+## Upcoming Excel 1.8 Event Release Features
+
+Below is the early thought of Event included in Excel JavaScript APIs. Further details about event argument design and sample code will be posted later. 
+
+1.	Data change event for Worksheet, Table with co-auth support
+
+    This type of event is triggered when there is data change happened, for example, cells value change on the grid.
+
+    |Object|Event| Description |
+    |:----|:----|:----|
+    | Worksheet    | onDataChannged | Occurs when data changed on specific worksheet |
```

Comment:
Please add periods after the descriptions here, and include the articles - i.e. on "a" specific worksheet.